chirp/src/db/content.rs

```diff
@@ -198,13 +203,114 @@ pub fn db_content_add(contents: Vec<FullContent>) -> Result<(), Box<dyn Error +
     Ok(())
 }
 
+pub fn db_content_add_words_phrases(
+    cc_id: i32,
+    cb: ContentBody,
+) -> Result<(), Box<dyn Error + Send + Sync>> {
+    let clean_text = strip_html_tags_from_string(&cb.body_text);
+    let phrases_tallies = collect_word_tallies_with_intersections(&clean_text);
+
+    let mut content_ids: Vec<i32> = Vec::new();
+    let mut phrases: Vec<String> = Vec::new();
+    let mut tallies: Vec<i32> = Vec::new();
+
+    for (phrase, tally) in phrases_tallies {
```

Comment:
Do we want all of the phrases? Maybe a phrase with a tally of 1 would be a mistake, like adding a URL?

Reply:
Well I think we want all phrases and should correct the phrase extraction so it doesn't make mistakes like adding URLs. That way phrasing can be used as intended for functionality such as article-to-article linking.